bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/ErrorValue.java

```diff
@@ -441,7 +442,7 @@ private Optional<StackTraceElement> filterStackTraceElement(StackTraceElement st
     }
 
     private String cleanupClassName(String className) {
-        return className.replace(GENERATE_OBJECT_CLASS_PREFIX, "");
+        return className.replace(GENERATE_OBJECT_CLASS_PREFIX, "").replaceAll(SPLIT_CLASS_SUFFIX_REGEX, "");
```

Comment:
If we generate split classes with object class prefix, we can get rid of this replace all.

Reply:
Discussed to keep `replaceAll` since it does not have an impact on the performance.